lib/README.md

```diff
@@ -295,3 +295,110 @@ if (false === $result) {
 ```
 
 This function returns the error message if any.
+
+### Class `WasmArrayBuffer`
+
+This class represents a buffer of bytes. It will be used to manipulate
+the memory of a WebAssembly instance.
+
+The class looks like this:
+
+``` php
+class WasmArrayBuffer
```

Comment:
`final`

Reply:
`WasmArrayBuffer` is already final, see https://github.com/wasmerio/php-ext-wasm/pull/35/files#diff-7b2e5dbe8736dd85c7c76bf99bf6c375R1890. It's an error in the documentation. I'm updating it. Thanks for the notice!